package.json

```diff
@@ -11,7 +11,7 @@
     "typescript": "tsc --project ./test/types/tsconfig.json",
     "test": "npm run lint && npm run unit && npm run typescript",
```

Comment:
Can you use yarn to run these as well?

Reply:
I'm trying to use npm as alias for yarn 😄

Reply:
Done